### ISSUE TITLE
Can't use on macos out of the box

### ISSUE THREAD

**Issue by dimaqq** [segevfiner/pybpf_asm#6]
I tried to `poetry add bpf-asm` to an empty project and it fails with:

```
  FAILED: bpf_asm/bpf_exp.yacc.c bpf_asm/bpf_exp.yacc.h /.../bpf_asm-0.1.2/_skbuild/macosx-13.0-arm64-3.11/cmake-build/bpf_asm/bpf_exp.yacc.c /.../bpf_asm-0.1.2/_skbuild/macosx-13.0-arm64-3.11/cmake-build/bpf_asm/bpf_exp.yacc.h
  cd /.../bpf_asm-0.1.2/_skbuild/macosx-13.0-arm64-3.11/cmake-build/bpf_asm && /usr/bin/bison -d -o bpf_exp.yacc.c /.../bpf_asm-0.1.2/bpf_asm/bpf_exp.y
  /.../bpf_asm-0.1.2/bpf_asm/bpf_exp.y:53.1-5: invalid directive: `%code'
  /.../bpf_asm-0.1.2/bpf_asm/bpf_exp.y:53.7-14: syntax error, unexpected identifier
```

I suspect that perhaps some system yacc/bison is getting used instead of brew-installed.

**Comment by segevfiner** [comment]
I think there is some trick I had to do to get it to use the right ones, need to check the GitHub Actions. Anyhow, I built wheels to help with that, I guess I might need to build wheels for Python 3.11 as well.

**Comment by dimaqq** [comment]
Wheels would be lovely!

**Comment by segevfiner** [comment]
https://github.com/segevfiner/pybpf_asm/blob/de8c8ee902a01502d4d1e88197bac45ecadb8bd9/.github/workflows/wheels.yml#L51

Seems like I had to add the Homebrew ones to the front of the PATH, guess I should also document this.

**Comment by dimaqq** [comment]
I've looked at that very code, tried that locally and that didn't work for me.
Maybe it's because `poetry add` does something extra, I'm not sure.
Edit: probably rather `cython` (also brew-installed) than poetry.

Then again, if a new set of wheels is built (macos 10.15 rather than 10.9) which includes arm64, the issue is gone for most users.

**Comment by segevfiner** [comment]
Released v0.1.3 with universal2 wheels, let me know if it doesn't work.
